spec/shr_oncology.txt

```diff
@@ -82,31 +85,295 @@ includes 0..1	BreastCancerHistologicGrade
 		Concept:		SCT#260869008 "Timing of stage"
 		Description:	"Indicates when the staging was done, in terms of treatment landmarks."
 		Value:			CodeableConcept	from StageTimingPrefixVS
+
+
+// ******* START OF MLT_Tumor_Mod ***********
+
+	EntryElement:	BreastCancerPrimaryTumorClassification
+	Based on:		CodedLaboratoryObservation
+	Concept:		SCT#78873005 "T category"
+	Description:	"The size and extent of the primary tumor, based on criteria defined by the staging system being used."
+					ObservationContext.ResultValue.CodeableConcept from TBD
+					LaboratoryObservationTopic.Focus value is type Identifier  // Lymph node identifier
+					LaboratoryObservationTopic.TopicCode is SCT#78873005 "T category"
+	0..0			LaboratoryObservationTopic.DiagnosticService  
+	0..0			LaboratoryObservationTopic.Specimen // also removed b/c this is determined lower
+	0..0			LaboratoryObservationTopic.Precondition
+	0..0			LaboratoryObservationTopic.FindingMethod
+			 		LaboratoryObservationTopic.EvaluationComponent
+	includes 0..1	StageSuffix
+					LaboratoryObservationTopic.PanelMembers.Observation
+	includes 0..1	TumorAssessment  // changed cardinality to 1 since this pertains only to a primary tumor.  For breast cancer, this is the largest size tumor.
+
+		Element:   		StageSuffix  
+		Based on:       CodedEvaluationComponent
+		Concept:        TBD#TBD 
+		Description:    "A suffix used in conjuction with certain cancer stages, based on criteria defined by the staging system being used."
+		Value:          CodeableConcept
+
 		
-		Element:		BreastCancerPrimaryTumorClassification
-		Concept:		SCT#78873005 "T category"
-		Based on:		CodedEvaluationComponent
-		Description:	"The size and extent of the primary tumor, based on criteria defined by the staging system being used."
-		Value:			CodeableConcept
-		0..1			StageSuffix
+		EntryElement:	TumorAssessment		// change to TumorAssessment
```

Comment:
TumorAssessment is not necessary now that the cardinality of PrimaryTumorClassification has 0..1.  Move TumorDimension and TumorMargin up one level.

Reply:
completed change on local branch. pending completion of group review to commit.